### ISSUE TITLE
direction on edges

### ISSUE THREAD

**Issue by alixander** [terrastruct/TALA#64]
<img width="571" alt="Screen Shot 2023-11-19 at 8 54 00 PM" src="https://github.com/terrastruct/TALA/assets/3120367/b2e3da13-dec6-4548-8f17-05ccd7ee47db">

TALA should be able to handle individual direction on edges. 

```d2
a -> b { direction: left }
```

We can ensure this.

**Comment by sbwiley** [comment]
Yes please. Any solutions I've seen from the community to give directional layout hints to TALA on a per-edge basis would be difficult to write/understand, esp. for diagrams that will have dozens or hundreds of nodes.
